workers/unity/Packages/com.improbable.gdk.core/Logging/LoggingUtils.cs

```diff
@@ -8,7 +9,7 @@ public class LoggingUtils
         public const string EntityId = "EntityId";
         public const string LoggerName = "LoggerName";
 
-        public static Collections.Option<EntityId> ExtractEntityId(Dictionary<string, object> data)
+        public static EntityId? ExtractEntityId(Dictionary<string, object> data)
```

Comment:
Why are we boxing these? Can't we normalize the inputs so we're just storing `EntityId`?

Reply:
cc  @martinzlocha I think the idea was the data is stored in a generic `string` -> `object` map. Since these are now value types.. it does box which is annoying